packages/syft/proto/lib/sympc/session.proto

```diff
@@ -7,12 +7,12 @@ import "proto/lib/python/dict.proto";
 
 
 message MPCSession {
-  bytes protocol = 8;
-  bytes uuid = 1;
+  string uuid = 1;
```

Comment:
I think using bytes object would be optimal  for uuid.

Reply:
Ahh...I remember why I used string. it is because we won't have to do a lot of conversions at the moment and it might be easier to debug until we reach a more "production" like code.

One example is that we keep rank -> session_uuid (in SyMPC) and that session_uuid is a string because it is easier for devs to "debug" it.